sdk/ai/azure-ai-inference/src/main/java/com/azure/ai/inference/ModelServiceVersion.java

```diff
@@ -13,7 +13,12 @@ public enum ModelServiceVersion implements ServiceVersion {
     /**
      * Enum value 2024-05-01-preview.
      */
-    V2024_05_01_PREVIEW("2024-05-01-preview");
+    V2024_05_01_PREVIEW("2024-05-01-preview"),
+
+    /**
+     * Enum value 2024-08-01-preview.
+     */
+    V2024_08_01_PREVIEW("2024-08-01-preview");
```

Comment:
Where did this API version come from? We only have one API version, 2024-05-01-preview.

Reply:
This API version is required for structured JSON output to work on the gpt-4o AOAI endpoints, otherwise I get a "not supported" error.